query-tuning/enabling_pg_hint_plan.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Enabling pg_hint_plan'
+backlink_href: /docs/query-tuning
+backlink_title: 'Query Tuning'
+---
+
+`pg_hint_plan` is a PostgreSQL extension that offers the possibility to tweak
+PostgreSQL execution plans using so-called "hints" in SQL comments.
+
+This can help you guide the planner to pick or avoid specific execution plans,
+which is useful for tuning queries.
+
+## Installing and enabling (Self-managed)
+
+For self-managed servers, install `pg_hint_plan` either by building a binary
+module from the code, or from a binary package for Debian or Ubuntu.
```

Comment:
Should we document `rpm`-based systems as well? It looks like the package there is called `pg_hint_plan_<postgres version>`.

Reply:
Sounds good, I updated and added that version too (also tested that it works)